src/components/Education/Education.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Education = () => {
+    return (
+        <div>
+            <h1>Hello, Education</h1>
```

Comment:
Reorganize these into pages folder.

[Reference](https://github.com/smaranjitghose/doc2pen/tree/master/src/pages)

Reply:
Will resolve tommorow,as I've to change all the files and thier names and paths